packages/astro/src/@types/astro.ts

```diff
@@ -2331,15 +2266,14 @@ export interface AstroUserConfig {
 		 * const labradorData = await getEntry('dogs', 'labrador-retriever');
 		 * ```
 		 *
-		 * #### Rendering content
-		 *
 		 * Entries generated from markdown or MDX can be rendered directly to a page using the `render()` function.
 		 *
 		 * :::caution
```

Comment:
```suggestion
		 * :::note
```
I think we can get away with a note here! Orange flames can look scary, and this isn't an e.g. security issue. Just a difference in notation!